src/ripple/test/jtx/Env.h

```diff
@@ -182,7 +182,7 @@ class Env
         JTx jt(std::forward<JsonValue>(jv));
         invoke(jt, fN...);
         autofill(jt);
-        return jt;
+        return std::move(jt);

```

Comment:
I'll undo this change. It was required for an earlier iteration before `JTx::props_` was made copyable.